### PR TITLE
Try to fix copying of files

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: tests/performance/onnxmodel/onnxmodel.rb [hunk 1]
```diff
@@ -40,7 +40,7 @@ def test_onnx_deploy
   def fetch_model
     @node = vespa.nodeproxies.first[1]
     file = @node.fetchfiles(:webhost => WEBHOST, :file => "pub/systemtests/performance/onnxmodel/ranking_model.onnx", :nocache => true, :nochecksum => true).first
```
Comment (baldersheim) [inline tests/performance/onnxmodel/onnxmodel.rb:42]: But can this test be here ? or is WEBHOST public ?

Comment (hmusum) [reply tests/performance/onnxmodel/onnxmodel.rb:42]: I think you are right, this needs to be an internal test

Comment (hmusum) [reply tests/performance/onnxmodel/onnxmodel.rb:42]: Or we could generate onnx file, but that would require some dependencies to be installed, need to look into that

Comment (jobergum) [reply tests/performance/onnxmodel/onnxmodel.rb:42]: It would be great to have this in the public - we can pull the model from our public S3 bucket?

Comment (hmusum) [reply tests/performance/onnxmodel/onnxmodel.rb]: Done

**Comment by hmusum** [comment]
@jobergum or @baldersheim please take a look

**Review by jobergum — APPROVED** [review]

**Pull request merged**